apps/expand.go

```diff
@@ -6,29 +6,67 @@ package apps
 type ExpandLevel string
 
 const (
-	ExpandDefault = ExpandLevel("")
-	ExpandNone    = ExpandLevel("none")
-	ExpandAll     = ExpandLevel("all")
-	ExpandSummary = ExpandLevel("summary")
+	ExpandDefault ExpandLevel = ""
+	ExpandNone    ExpandLevel = "none"
+	ExpandAll     ExpandLevel = "all"
+	ExpandSummary ExpandLevel = "summary"
 )
 
+// Expand is a clause in the Call struct that controls what additional
+// information is to be provided in each request made.
+//
+// By default only the IDs of certain entities are provided in the request's
+// Context. Expand allows to selectively add data to ExpandedContext, including
+// privileged information such as access tokens, and detailed data on Mattermost
+// entities, such as users and channels.
+//
+// Based on the app's GrantedPermissions, Bot, User, or Admin-level tokens may
+// be provided in the request. If the app connects to a 3rd party, it may store
+// authentication data in the Mattermost token store and get the token data
+// expanded in the request.
+//
+// When expanding Mattermost data entities, the apps proxy must not exceed the
+// highest available access level in the request's Context.
 type Expand struct {
-	App        ExpandLevel `json:"app,omitempty"`
+	// App: all. Details about the installed record of the App. Of relevance to
+	// the app may be the version, and the Bot account details.
+	App ExpandLevel `json:"app,omitempty"`
+
+	// ActingUser: all for the entire model.User, summary for BotDescription,
+	// DeleteAt, Email, FirstName, Id, IsBot, LastName, Locale, Nickname, Roles,
+	// Timezone, Username.
```

Comment:
If I'm not mistaken, right now we are also passing the password field (salted I guess/hope). Should we sanitize more here?